HULKs/hulk
# Improve fall motion

## Introduced Changes

When a NAO is falling, it currently doesn't do much except reducing the stiffness and moving the head slightly.

The new falling motion contains bending the knees and unrolling. This should reduce the strain on impact.


## Ideas for Next Iterations (Not This PR)

Create the motion such that it brings the NAO into a position more favourable for the following stand-up motion.


## How to Test

Test in Webots: Start Webots with `webots/worlds/falling.wbt` and start the simulation. Robot should then do the motion in the next 5s.

Test in Lab with Robot: Push the robot. Moreover it was tested in a test game.
